### PR TITLE
Feature/#81 sentences wip

### PR DESCRIPTION
**Pull request by manudous**
@brauliodiez Step 3 review

### CODE REVIEW

File: src/pods/test-sentences/components/span.component.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { SentenceEntityVm } from '../test.sentences.vm';
+import * as styles from 'common/styles/tests.styles';
+
+interface Props {
+  sentenceSelected: SentenceEntityVm;
+  verbForms: string;
+  rightAnswerValue: boolean;
+}
+
+export const SpanComponent: React.FunctionComponent<Props> = props => {
```
Comment (brauliodiez) [inline src/pods/test-sentences/components/span.component.tsx:12]: SpanComponent is too generic, let's use something more related to the domain, e.g.

SentenceComponent

Comment (manudous) [reply src/pods/test-sentences/components/span.component.tsx]: done

**Comment by manudous** [comment]
I need you check BodyComponent because I had to put the enum there, but to reference it in the SentenceComponent I have to use the enum, that is, I get to SentenceComponent as 0 1 2 if it could be put in another way to make it more descriptive.

**Comment by manudous** [comment]
Done.

File: src/pods/test-sentences/components/body.component.tsx [hunk 2]
```diff
@@ -41,17 +53,23 @@ export const BodyComponent: React.FunctionComponent<Props> = props => {
...
+              {present}
+            </Button>
+            <Button onClick={handleButtonValue} value="Past">
+              {past}
+            </Button>
+            <Button onClick={handleButtonValue} value="Participle">
+              {participle}
+            </Button>
           </ButtonGroup>
         </div>
```
Comment (brauliodiez) [inline src/pods/test-sentences/components/body.component.tsx:66]: Enclose all this Button Group in a component, by doing this you could get this component as simple as:

```tsx
  return (
    <main className={mainContainer}>
      <h1 className={title}>{translation.toUpperCase()}</h1>
      <div className={backContainer}>
        <div className={pictureContainer}>
          <img className={picture} src={`/assets/verb-images/buy.png`} />
        </div>
        <div className={buttonGroupContainer}>
           <TensesButtonGroup/>
        </div>
        <div>
          <SentenceComponent
            sentenceSelected={sentenceSelected}
            rightAnswerValue={rightAnswerValue}
            verbForms={verbForms}
          />
        </div>
      </div>
    </main>
```

Tenses buttonGroup will have the proper callbacks and props

**Pull request merged**